dashboard/app/models/census/README.md

```diff
@@ -62,7 +62,7 @@ The census_submissions table is replicated into Redshift via [dms task](https://
 
 # Historic Notes
 
-Before 2022, census summaries were computed via a nightly [cron job](https://github.com/code-dot-org/code-dot-org/blob/80777d646a9351de59404fbd173c67799c43dbda/bin/cron/update_census_mapbox) by an algorithm maintained by the engineering team. Inputs to this algorithm were stored in the ap_cs_offerings, ap_school_codes, ib_cs_offerings, ib_school_codes and state_cs_offerings tables. 
+Before 2022, census summaries were computed via a nightly [cron job](https://github.com/code-dot-org/code-dot-org/blob/80777d646a9351de59404fbd173c67799c43dbda/bin/cron/update_census_mapbox) by an algorithm maintained by the engineering team. Inputs to this algorithm were stored in the ap_cs_offerings, ap_school_codes, ib_cs_offerings, and ib_school_codes tables. 
```

Comment:
please add this back, this is a description of the old implementation and it will be more accurate in its original form.